src/components/Tokens/TokenRowWallets.tsx

```diff
@@ -217,19 +217,28 @@ export const TokenRowWallets = component$<TokenRowWalletsProps>(
         .value;
       latestTokenPrice.value = (await data.next()).value["price"];
 
-      for await (const value of data) {
-        if (value.action === "CREATE") {
-          if (value.type === "BALANCE") {
-            currentBalanceOfToken.value = convertWeiToQuantity(
-              value.result["walletValue"],
-              parseInt(decimals),
-            );
+      let timeOfPreviousBalanceChange = new Date(
+        "1970-01-01T00:00:00Z",
```

Comment:
Can we use 0 instead of this tring? ig it would be the same  so we would get unix expoch start, and less cumbersome than a string

Reply:
i guess we have to change the whole expression new Date("1970-01-01T00:00:00Z").getTime(); will return 0 because it returns time in ms